frontend/src/components/Leaderboard.tsx

```
@@ -1,7 +1,14 @@
 import styles from "../styles/Leaderboard.module.css";
 import { useEffect, useState } from "react";
 import UserRankCard from "./UserRankCard";
-import { getLeaderboard, getLeaderboardDaysLeft } from "@/api/api";
+import {
+    getLeaderboard,
+    getLeaderboardDaysLeft,
+    getLeaderboardPreference,
+    setLeaderboardPreference,
+} from "@/api/api";
+import { get } from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)